core/src/main/java/com/linecorp/armeria/client/endpoint/dns/DnsEndpointGroupBuilder.java

```diff
@@ -153,6 +157,29 @@ final DefaultDnsResolver buildResolver(Consumer<DnsNameResolverBuilder> customiz
                                      hostsFileEntriesResolver());
     }
 
+    /**
+     * Adds the {@link DnsQuestionListener}s that listens to the result of querying {@link DnsRecord}s.
```

Comment:
Should we add the comment to other Javadocs as well?
```suggestion
     * Adds the {@link DnsQuestionListener}s that listens to the result of querying {@link DnsRecord}s.
     * If no {@link DnsQuestionListener} is configured, {@link DnsQuestionListener#of()}
     * is used by default
```